docs/user-guide/kubectl-conventions.md

```diff
@@ -35,7 +35,8 @@ In order for `kubectl run` to satisfy infrastructure as code:
 
 * Pod - use `run-pod/v1`.
 * Replication controller - use `run/v1`.
-* Deployment - use `deployment/v1beta1`.
+* Deployment, using `extensions/v1beta1` endpoint - use `deployment/v1beta1`.
+* Deployment, using `apps/v1beta1` endpoint - use `deployment/apps.v1beta1`.
```

Comment:
But we're not deprecating this in 1.6, so it doesn't make sense to say so, yet. When we do we'll add appropriate info. But added both default and recommended.